.github/workflows/npmpublish.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
@imagekitio please sync changes of [imagekit-developer:master](https://github.com/imagekit-developer/imagekit-nodejs/tree/master) --into-> [imagekit-developer:dev](https://github.com/imagekit-developer/imagekit-nodejs/tree/dev)